numpy/distutils/command/config.py

```diff
@@ -64,7 +64,7 @@ def _check_compiler (self):
                         % (e, self.compiler.__class__.__name__)
                     print(textwrap.dedent("""\
                         ============================================================================"""))
-                    raise distutils.errors.DistutilsPlatformError(msg)
+                    raise distutils.errors.DistutilsPlatformError(msg) from None
```

Comment:
I think `from e` would make more sense here - while the message does contain the "original exception", it does not actually show the likely relevant trace.

Reply:
Okay, I did that because the `msg` already contained some info and I thought it was enough. but I'll update this.
thanks. :)

Reply:
See my comment below - it seems distutils is full of this pattern, and I'm not really sure what the right thing to do is.